qemu/tests/virtio_fs_guest_users_allowed_by_group_permission.py

```diff
@@ -0,0 +1,116 @@
+import os
+import shutil
+
+from virttest import nfs, data_dir
+from virttest import env_process
+from virttest import error_context, utils_disk, utils_misc
+
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Grant write access at users allowed by group permission
+    on shared dir( linux only )
+
+    1) Run the virtiofsd daemon on the host and Set log level to "debug".
+    2) Boot a guest on the host.
+    3) Log into guest with root, then mount the file system.
+    4) In the guest, add a new user(u1)
+       and set the user to the wheel group (as supplementary group)
+    5) In guest, using root to create a test directory at the shared dir.
+    6) Run the basic io test with the u1 user
+    7) After test, clear the environment.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    add_user_cmd = params.get("add_user_cmd")
+    del_user_cmd = params.get("del_user_cmd")
+    username = params.get("new_guest_user")
+    fs_dest = params.get("fs_dest")
+    fs_target = params.get("fs_target")
+    setup_local_nfs = params.get("setup_local_nfs", "no") == "yes"
+
+    testdir = "testdir"
+    guest_root_session = None
+    guest_user_session = None
+    vm = None
+    nfs_local = None
+
+    try:
+        # create the share dir before daemon running
```

Comment:
No need to customize virtiofsd parameters, just setting   fs_binary_extra_options +=  " --log-level debug" is enough, avocado will start virtiofsd according during start vm.